pinot-server/src/main/java/org/apache/pinot/server/starter/helix/BaseServerStarter.java

```diff
@@ -945,4 +951,22 @@ private void initSegmentFetcher(PinotConfiguration config)
   protected AdminApiApplication createServerAdminApp() {
     return new AdminApiApplication(_serverInstance, _accessControlFactory, _serverConf);
   }
+
+  private void configureGroovySecurity() {
+    GroovyStaticAnalyzerConfig config = null;
+    try {
+      GroovySecurityConfigManager manager = new GroovySecurityConfigManager(_helixManager);
+      config = manager.getConfig();
+
+      if (config == null) {
+        config = GroovyStaticAnalyzerConfig.createDefault(true);
+        manager.setConfig(config);
+      }
+    } catch (Exception _ex) {
+      LOGGER.error("Failed to read config from ZK. Loading Default configuration.");
```

Comment:
Directly throw exception instead of logging error. We want to fail the server start if security is not guaranteed